### ISSUE TITLE
HW1 Ship Design for Speed/Health

### ISSUE THREAD

**Issue by cloaked2222** [HW-PlayersPatch/2.3#55]
Re-posting as its own thread.

The basic ship properties are build costs, times, ship speed, and health. Getting these right requires a good top-down design. Generally these should have a similar feel to the classic games, while being balanced for HWR.

As part of the health review I decided to remove all the sidearmour damage less than 1.0, and adjust health accordingly. Gearbox did this as a quazi health buff which made little sense.... The only sidearmour damage left now is the 1.2x on all frigs, which is a relic design that should stay in place.

**Health Review**
![image](https://user-images.githubusercontent.com/18016573/41816812-6e1db628-775c-11e8-8c00-d1358f1f37e2.png)
![image](https://user-images.githubusercontent.com/18016573/41816869-add52f2a-775d-11e8-9d71-801bae62bab6.png)
![image](https://user-images.githubusercontent.com/18016573/41816871-b88676fe-775d-11e8-9137-49085e6981b7.png)
![image](https://user-images.githubusercontent.com/18016573/41816874-c301d038-775d-11e8-8f39-6e9a95fe6d26.png)
hw1 MS health is set between vay/hiig when upgraded.
![image](https://user-images.githubusercontent.com/18016573/41816882-e5570342-775d-11e8-8ca2-d69ebfdd7f23.png)

I'll document a review on speed later. I already did one last month, I just didn't document it in a nice spreadsheet... b7 speeds are pretty good, with some planned changes for b8. Namely, hw1 colls, refineries, cloaked fighters, corvs, specialty frigs, and cloak gens, etc.

HW1 Ideal Design.xlsx spreadsheet is here:
https://github.com/HW-PlayersPatch/2.3/tree/Cloaked/Balance%20Info

**Comment by cloaked2222** [comment]
Most of the speed changes have already been discussed before, or were in my June 2nd files.

**Speed Review**
b7 Speed | b7 Upgraded | Ratio |
b8 Speed | Ratio |
hw1c Speed | Ratio
![image](https://user-images.githubusercontent.com/18016573/41888354-12d2599a-78d3-11e8-8d74-d124ae55a462.png)
![image](https://user-images.githubusercontent.com/18016573/41888368-20192c82-78d3-11e8-9d55-f98819ab31bd.png)
![image](https://user-images.githubusercontent.com/18016573/41888385-28de2944-78d3-11e8-9ae3-cb64630963dc.png)
![image](https://user-images.githubusercontent.com/18016573/41888394-32011496-78d3-11e8-8ec1-885d3187544d.png)
![image](https://user-images.githubusercontent.com/18016573/41888401-3ecee004-78d3-11e8-8ecf-164ad9106c13.png)

Its very nice to have all these reviews finally documented. In the past we basically just did these type of reviews on a ship by ship basis when there were problems... Things would have been so much easier is this was done by gbx from the start...

**Comment by Novaras** [comment]
Cloaked fighters are better at 420 as I had them although it's a tiny difference I guess. Harrass this powerful shouldn't allow hamfisted blunders being forgiven by a high speed stat, that was my thought.

Repair corvettes should be as fast as multiguns

**Comment by Echo1707** [comment]
Support frigates are **almost** as fast as heavy corvettes. I stopped liking their speed when Fear kited the support frigate I was focusing down with my heavy corvettes to the end of the map :(

I also don't like very much the upgraded speeds of Vaygr's infiltration frigates: they are going as fast as corvettes, and even faster than salvettes.

**Comment by Novaras** [comment]
Frigates have always been able to kite out vettes to varying degrees... For example torps lose hard to lasers normally but can actually perform quite reasonably when kiting.

I'm always in the camp that combat utility should be faster than their target audience anyway.

**Comment by Echo1707** [comment]
Support frigates are not anti-corvettes, so kiting doesn't make them kill more corvettes. The target audience for support frigates should be frigates, not corvettes, and I don't have an issue with support frigs being the fastest HW1 frigs. But not even in HW1C support frigates where as fast as heavy corvettes.

Looking at the numbers, in HW1C assault frigates had 325 speed, support frigates 450 and heavies 520.
If we say the slowest has 1 speed, then suports are at 450/325 (=1.38) and heavies at 1.6.
If in build 8 heavy corvettes stays at 290 and assault frigates at 206, then with a proportion we get:
`(290-206) : 1.6 = (x-206) : (450/325)`
`1.6 * (x-206) = (290-206) * (450/325)`
`x-206 = 84 * (450/325) / 1.6`
`x = 206 + 84*450/(325*1.6)
`x = 206 + 24300/520`
`x = 206 + 46.7`
`x = 252.7`

which is still faster than any other frigate (bare the infiltration frigate, which I have an issue with because it's going as fast as a corvette) and pretty similar to salvage corvettes and minelayers (255 speed), while in HW1C these corvettes had a speed of 425 (supports being at 450). How does 252.7 sound?

**Comment by Novaras** [comment]
I'm not strongly against it but I wouldn't personally make a change like that... I enjoy being able to run supports, if both of you dislike them being speedy it's fine I guess

**Comment by cloaked2222** [comment]
Its quite funny that support frigates ended up faster than repair corvettes in v2.1...

Spreadsheets are mostly class based, with some consideration for classes above/below. They're definitely not perfect. Based on your logic, ya support frigates could prolly come down to 260-270.

I'd guess scole made them fast, so they could support a corvette in a wall formation (popular strategy in hw1c). Prolly went with 287 since that's the hw2c vay infiltrator top speed. HWR's repair logic isn't as good as hw1c, so repair units likely need to be faster than hw1c. With corvette's speed going up, we may just want to leave support frigates as is, and potentially buff repair corvette speed to 320. This may be one of those cases where we can't really align with hw1c, due to HWR engine differences.

One this scole did really good, was tap into his artist skills. He would watch battles in detail, and tune things so they looked good, and worked together as intended. Its a shame really that the 666 bug destroyed all the work he put into formations.

**Comment by cloaked2222** [comment]
Fear: "Light vettes at 320 is nice and makes sense.
Don't know if multis also need the speed, light vettes are losing their identity..."
Still feel this way? From many people's perspectives, multi's were the fastest hw1 vette for 16+ years till gbx screwed them up. =P

Fear: "Iirc multigun HP change was very intentional and due to them overperforming in matchups as well as taking far too long to bring down... 1000 HP might be nice but 1400 is insanely high."
I haven't seen your tests so i dunno specifics. b7 MG 1100hp is still a lot less than heavies at 1700hp. Lowering their HP will make pulsar/missiles bring them down that much faster. Not too concerned if it takes ints forever to bring down multis.

Fear: "Fairly ambivalent about minelayer change, 270 was more like a QOL change."
haha, no comment :)

**Comment by Novaras** [comment]
Before I nerfed their penetration vs corvettes, multis were beating missile vettes 1-1 lol 

This is a pretty good example of why we should have playtested v8 before starting all this v9 stuff...

**Comment by Echo1707** [comment]
Test data for multiguns vs ints is here: https://github.com/HW-PlayersPatch/2.3/blob/Cloaked/TestData/hw1%20multiguns.xlsx

**Comment by cloaked2222** [comment]
Implemented:
https://github.com/HW-PlayersPatch/2.3/commit/f652c0daada6ccf28a5cede0e53e0e43eb83024d

Made a few tweaks from the OP, based on testing and discussion.
